VideoLocker/src/main/java/org/edx/mobile/view/adapters/InfiniteScrollUtils.java

```diff
@@ -100,8 +101,17 @@ private void onLoadMore() {
             adapter.setProgressVisible(true);
             pageLoader.loadNextPage(new PageLoadCallback<T>() {
                 @Override
+                public void onPartialPageLoaded(List<T> newItems) {
+                    if (isResetCalled()) {
+                        return; // Disregard result, since reset() was called
+                    }
+                    adapter.addAll(newItems);
+                    adapter.setProgressVisible(false);
```

Comment:
The partial load callback should not be changing the progress loader visibility state, so this line should be removed.